README.md

```diff
@@ -95,74 +95,48 @@ We need to do two things:
-  token: ''
-})
+const spawnBackend = init({ dev: true })
 ```
 
-The `init` function here takes three arguments:
+When developing locally with the Jamsocket Dev CLI, we can just pass `{ dev: true }` to the `init()` function. We'll replace this with account and service names and an API token when it comes time to deploy this to Jamsocket. You can see an example in [the `README` for the `@jamsocket/javascript` package](https://github.com/drifting-in-space/jamsocket-javascript).
 
-* an account name - that's the account name you created when you signed up for Jamsocket (which you can find on [the Settings page](https://app.jamsocket.com/settings) in case you forgot)
-* a service name - that's the name of the service we just created - `whiteboard-demo`
-* an api token - we can leave this blank for now as the Jamsocket dev CLI will take care of authentication while we're development. When it's time to go to production, we can get an API token from [the Settings page](https://app.jamsocket.com/settings).
+The `init()` call returns a `spawnBackend()` function that we'll use to, well, spawn a backend. It takes a single, optional `spawnOptions` argument. These `spawnOptions` are just camel-cased versions of the options accepted by the HTTP spawn API. (Our docs have more information about [spawn options for the HTTP API](https://docs.jamsocket.com/platform/reference/#spawn-a-service).) For now, we will only use one of those spawn options: `lock`. You can learn more about spawning with locks [here](https://docs.jamsocket.com/concepts/locks), but for now it suffices to say that we'll just use a document name. And for this demo, we'll just have one document that everybody edits called `whiteboard-demo/default`.
 
-And it returns a `spawnBackend()` function that we'll use to, well, spawn a backend. It takes a single, optional `spawnOptions` argument. These `spawnOptions` are just camel-cased versions of the options accepted by the HTTP spawn API. (Our docs have more information about [spawn options for the HTTP API](https://docs.jamsocket.com/platform/reference#spawn-a-service).) For now, we will only use one of those spawn options: `lock`. You can learn more about spawning with locks [here](https://docs.jamsocket.com/concepts/locks), but for now it suffices to say that we'll just use a document name. And for this demo, we'll just have one document that everybody edits called `whiteboard-demo/default`.
+The result of the `spawnBackend()` function contains a URL that you can use to connect to the session backend, a status URL which returns the current status of the session backend, and some other values like the backend's name and an optional bearer token which is useful when authenticating client requests to a session backend. (We aren't using these bearer tokens in this demo, but you can learn more about them [here](https://docs.jamsocket.com/concepts/auth-with-backend/)).
```

Comment:
the first sentence is a little long, maybe break it up or at least add a comma "optional bearer token, which"?